docs/configuration/namespaces.md

```
@@ -0,0 +1,429 @@
+metadata:
+  name: argocd-agent-params
+  namespace: argocd
+data:
+  principal.allowed-namespaces: "agent-prod,agent-staging,agent-dev"
+```
+
+### Multiple Argo CD Instances on the Control Plane
+
+!!! warning "Namespace Conflicts with Other Argo CD Instances"
+
+    When running more than one Argo CD instance on the control plane cluster, special care must be taken if the other instances also use `application.namespaces` (apps-in-any-namespace).
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)